api/src/services/listing.service.ts

```diff
@@ -1016,6 +1016,7 @@ export class ListingService implements OnModuleInit {
     const newListingData: ListingCreate = {
       ...mappedListing,
       name: dto.name,
+      assets: [],
```

Comment:
Discussed a BE error I was running into with Eric and since it only required this one tweak, we agreed to include that here to not block this PR(cc @mcgarrye)